openshift/4.0/templates/jobs/db-deploy.yaml

```diff
@@ -65,3 +65,9 @@ parameters:
   - name: NAMESPACE
     description: the openshift namespace to run this job in.
     required: true
+  - name: GIT_BRANCH
+    description: the git branch to pull the database scripts from.
+    required: true
+  - name: DB_SECRET_NAME
+    description: the git branch to pull the database scripts from.
+    required: true
```

Comment:
Question - I'm assuming the corresponding secrets have been updated/configured in OCP?